socksio/_types.py

```diff
@@ -0,0 +1,3 @@
+import typing as t
```

Comment:
nit: the alias to `t` isn't doing much I don't think?

Reply:
Not really, no 👍